src/components/SingleArticle.jsx

```diff
@@ -56,7 +58,8 @@ export default function SingleArticle({users, setUsers}) {
                 <h3>Author: {article.author}p</h3>
                 <h4>Topic: {article.topic}</h4>
                 <p>{article.body}</p>
-                < Comments users={users} setUsers={setUsers}/>
+                <InsertComment article={article} username={username} isNewData={isNewData} setIsNewData={setIsNewData}/>
```

Comment:
if article.article_id is the only value which is going to be used in InsertComment, would be better to only pass the id down instead of passing the whole object